### DISCUSSION TITLE
Best Learning Material for KoliBri?

### DISCUSSION THREAD

**Discussion by josor001** [public-ui/kolibri#6323] [Q&A]
Hallo zusammen,

bin jetzt beruflich schon öfter über KoliBri gestolpert und wollte mich nun gerne mal reinlesen. Habt ihr Empfehlungen für Lernmaterial bzw. gibt es nen Kurs o.ä.? Repo anschauen und die GitHub Page ist klar, aber ich dachte, dass es vielleicht noch was praktisches gibt, das ich nicht sofort gefunden habe :)!

**Comment by deleonio** [comment]
Hi @josor001,

welchen Anwendungsfokus hast Du? Für die App-Entwicklung könnten folgenden Samples interessant sein:

- https://github.com/public-ui/kolibri/tree/develop/packages/samples/react/src
- https://github.com/public-ui/a11y-standardanforderungskatalog
